rails/etsy_product_catalog/app/views/listings/show.html.erb

```diff
@@ -1,12 +1,13 @@
 <h1>Product details</h1>
 
 
-    <h2><%= @listing.title %></h2>
-    <p><%= @listing.description %></p>
+<h2><%= @listing.title %></h2>
+<p><%= @listing.description %></p>
+<%= image_tag("#{@listing.main_image}") %>
```

Comment:
minor point - assuming @listing.main_image is a string, interpolating it is unnecessary. Simply image_tag(@listing.main_image) will work.